Assets/MixedRealityToolkit/_Core/Utilities/PlatformUtility.cs

```diff
@@ -8,59 +8,78 @@ namespace Microsoft.MixedReality.Toolkit.Core.Utilities
 {
     public static class PlatformUtility
     {
-        public static bool IsPlatformSupported(this RuntimePlatform runtimePlatform, SupportedPlatforms platform)
+        public static bool IsPlatformSupported(this RuntimePlatform runtimePlatform, SupportedPlatforms platforms)
         {
-            if (platform == (SupportedPlatforms)(-1))
-            {
-                return true;
-            }
+            SupportedPlatforms target = GetSupportedPlatformMask(runtimePlatform);
+            return IsPlatformSupported(target, platforms);
+        }
+
+        private static SupportedPlatforms GetSupportedPlatformMask(RuntimePlatform runtimePlatform)
+        {
+            SupportedPlatforms supportedPlatforms = 0;
 
             switch (runtimePlatform)
             {
                 case RuntimePlatform.WindowsPlayer:
                 case RuntimePlatform.WindowsEditor:
-                    return (platform & SupportedPlatforms.WindowsStandalone) != 0;
+                    supportedPlatforms |= SupportedPlatforms.WindowsStandalone;
```

Comment:
Is there ever a case where a runtimeplatform maps to more than one supported platform? If not, why not just a return without a local variable here?

Reply:
No, unity only ever has one platform set at a time.